### PR TITLE
Few more components added, and refactor CSS code.

### PR DESCRIPTION
**Pull request by saurabhchirde**
1. Color page updated with the new layout.
2. Name changed to Pabble-UI.
2. Fixed navigation for mobile (hamburger).
3. New page added:  Getting started, Typography. 
4. Some new cards were added as per the requirement of the e-commerce layout. (text overlay cards and information cards), 
5. New input fields are created (slider, radio, checkbox, color, and search).
6. Refactored and created separate CSS files for navigation, typography, component overview, etc.

### CODE REVIEW

**Review by j24m — APPROVED** [review]

**Review by m26unkwn — APPROVED** [review]
good to merge. Saurabh

File: docs.html [hunk 1]
```diff
@@ -20,104 +21,195 @@
...
+      <!-- font icons -->
 
+      <h2 id="font-icons" class="mg-4-top mg-1-bot title-lg-wt-5">Font icons</h2>
+      <p class="mg-1point4-tb title-md-wt-4">To use the Icon component, you must first add the fontawesome icons. Copy
+        and paste the below link in the '&lt;head&gt;' tag of your html, just after pasting above CSS link and you are
+        done with the installation process!
+        Now you can use any component just by adding their respective class/given code.</p>
 
-    <section id="getting-started" class="pd-1-lr pd-2-bot mg-3-top">
+      <script style="height: 100%" type="text/plain" class="language-html code-get-started">
```
Comment (SalonSikdar) [inline docs.html:202]: Avoid inline styling.

File: docs.html [hunk 2]
```diff
@@ -20,104 +21,195 @@
...
-        <h2 class="nav-card nav-btn">Cards</h2>
-      </a>
-      <a href="./Documentation/docpage/image.html">
-        <h2 class="nav-img nav-btn">Image</h2>
-      </a>
-      <a href="./Documentation/docpage/input.html">
-        <h2 class="nav-input nav-btn">Input</h2>
-      </a>
+    <a href="./Documentation/docpage/alert.html">
+      <h1 class="nav-alert nav-btn">Alerts</h1>
```
Comment (SalonSikdar) [inline docs.html:74]: For class names of items in the nav list, are different style classes needed? please revisit this, unless you're adding a custom style to each list item, 
"nav-alert, nav-button, nav-badge" etc

File: Documentation/docpage/about.html [hunk 3]
```diff
@@ -20,74 +20,129 @@
...
+      <h1 class="nav-card nav-btn">Cards</h1>
+    </a>
+    <a href="./image.html">
+      <h1 class="nav-img nav-btn">Image</h1>
+    </a>
+    <a href="./input.html">
+      <h1 class="nav-input nav-btn">Input</h1>
+    </a>
+    <div class="connect-mobile-view">
+      <hr class="nav-break-line">
```
Comment (SalonSikdar) [inline Documentation/docpage/about.html:137]: "nav-line-break" sounds more verbose

**Review by SalonSikdar — APPROVED** [review]

File: docs.html (hunk 1 above)
Comment (saurabhchirde) [reply docs.html:202]: These inline are used, only on those snippets that required less height.

File: docs.html (hunk 2 above)
Comment (saurabhchirde) [reply docs.html:74]: Thank you for pointing this out, I had already applied styles on them using the child selector. So removed both of them.

**Pull request merged**